hack/devtools/rp_dev_helper.sh

```diff
@@ -0,0 +1,448 @@
+        if  [[ -z "$arg" ]];  then
+            echo "Error: an empty var. Please try again"
+            return 1
+        fi
+    done
+
+    local return_line=$(grep -A 2 "func $1" "$2" | grep 'return')
+    echo "$return_line" | sed 's/.*"\(.*\)@sha256.*/\1/'
+}
+
+VERSION_CONST_FILE="pkg/util/version/const.go"
+# Function to get image name and tag
+get_digest_tag() {
+    if [[ $# -ne 1 ]]; then        
+        echo "Usage $0 <IMAGE_NAME>. Please try again"
+        exit 1
+    fi
+    local IMAGE_NAME=$1
+    local IMAGE_TAG=$(extract_image_tag "$IMAGE_NAME" "$VERSION_CONST_FILE")
+    echo $IMAGE_TAG
+}
+
+# Function to copy image using skopeo
+copy_digest_tag() {
+    # Check if exactly non-empty four arguments are provided
+    if [[ $# -ne 4 ]]; then        
+        echo "Usage $0 <PULL_SECRET> <SRC_ACR_NAME> <DST_ACR_NAME> <IMAGE_TAG>. Please try again"
+        exit 1
+    fi
+    for arg in "$1" "$2" "$3" "$4"; do
+        if  [[ -z "$arg" ]];  then
+            echo "Error: an empty var. Please try again"
+            exit 1
+        fi
+    done
+    echo "INFO: Copy image from one ACR to another ..."
+    local PULL_SECRET=$1
+    local SRC_ACR_NAME=$2
+    local DST_ACR_NAME=$3
+    local IMAGE_TAG=$4
+    
+    SRC_AUTH=$(echo "$PULL_SECRET" | jq -r '.auths["'$SRC_ACR_NAME'.azurecr.io"].auth' | base64 -d)
+    DST_TOKEN=$(az acr login -n "$DST_ACR_NAME" --expose-token | jq -r .accessToken)
+    
+    skopeo copy \
+        --src-creds "$SRC_AUTH" \
+        --dest-creds "00000000-0000-0000-0000-000000000000:$DST_TOKEN" \
+        "docker://$SRC_ACR_NAME.azurecr.io$IMAGE_TAG" \
+        "docker://$DST_ACR_NAME.azurecr.io$IMAGE_TAG"
+}
+
+check_acr_repo() {
+    # Check if exactly two non-empty arguments are provided
+    if [[ $# -ne 2 ]]; then
+        echo "Usage: $0 <ACR_Name> <Repository>"
+        exit 1
+    fi
+
+    # Don't skip deployment creation when SKIP_DEPLOYMENTS is set to "false" 
+    if [[ "${SKIP_DEPLOYMENTS}" == "false" ]]; then
+        echo "'SKIP_DEPLOYMENTS' env var is set to false. Don't skip ACR repo mirroring for repository $2."
+        return 1
+    fi
+
+    # Get the repository tag
+    repo_tag=$(az acr repository show-tags --name "$1" --repository "$2" -o tsv | tr '\n' ' ')
+
+    # Check if the repository tag is empty
+    if [[ -n "$repo_tag" ]]; then
+        echo "Repository '$2' in ACR '$1' exists with tag '${repo_tag}'."
+        return 0
+    else
+        echo "Repository '$2' doesn't exist in ACR '$1'."
+        return 1
+    fi
+}
+
+# "openshift-release-dev/ocp-release" "openshift-release-dev/ocp-v4.0-art-dev" were excluded as they don't include an image/tag
+acr_repos=("azure-cli" "rhel8/support-tools" "rhel9/support-tools" "openshift4/ose-tools-rhel8" "ubi8/ubi-minimal" "ubi9/ubi-minimal" \
+        "ubi8/nodejs-18" "ubi8/go-toolset" "app-sre/managed-upgrade-operator" "app-sre/hive" "distroless/genevamdm" "distroless/genevamdsd" \
+        "aro" "fluentbit")
```

Comment:
I was using it for easier viewing of the array values (without expanding the IDE/browser to see all the values). If that's still not necessary, then I will remove that.

Reply:
You can leave the line breaks, there's just no need to append `/` for each line break.
Usually I prefer to seperate each array item with a line break, but there are so many here, it would take up a lot of page space.